solidity/test/bridge/Bridge.Frauds.test.ts

```diff
@@ -530,6 +531,283 @@ describe("Bridge - Fraud", () => {
     )
   })
 
+  describe("defeatFraudChallengeWithHeartbeat", () => {
+    let heartbeatWalletPublicKey
+    let heartbeatWalletPublicKeyHash
+    let heartbeatWalletSigningKey
```

Comment:
Could we give these variables types?

Reply:
Good call. See f05032c32d88e1e42c32a4cb109ac63d0ca8b163.